modules/tinymce/src/plugins/quickbars/test/ts/browser/KeyboardShortcutContextualMenuTest.ts

```diff
@@ -0,0 +1,33 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinySelections, TinyContentActions, McEditor, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import QuickbarsPlugin from 'tinymce/plugins/quickbars/Plugin';
```

Comment:
We normally only call this `Plugin` if it's the test for the plugin and we don't have multiple plugin imports so can you please rename this?

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/8452b1943ab4421ba45b9bd4555fa17ea3396abd